tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -346,6 +349,9 @@ const exportContacts = async () => {
   await (await exportButton()).click();
 };
 
+const cardFieldLabelText = async (label) => await (await $(`.cell.${label} label`)).getText();
```

Comment:
@lorerod @ngaruko similar to my previous comment, would it be better to have 1 function returning 1 object that has label and value? That can make the test cleaner
```
return { 
label: await (await $(`.cell.${label} label`)).getText() 
value: await (await $(`.cell.${label} p`)).getText()
}
```

Reply:
@latin-panda idea is to anticipate needing this information in another test scenario. When that happens, we will have to create the function, use it and change this code so that it also uses it. Creating the function right now that we need the info is better.